### PR TITLE
remove /rise25/ and redirect to external page

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary
Redirect from `/rise25/`to `https://rise25.mozilla.org

## Significant changes and points to review
- Removed all old rise25 assets and templates 
- set up redirects 
- added tests in map_globalconf.py

## Issue / Bugzilla link
#13754 

## Testing
`py.test --base-url http://localhost:8000 tests/redirects/`

### CODE REVIEW

File: bedrock/mozorg/redirects.py [hunk 1]
```diff
@@ -569,4 +569,7 @@ def decider(request, **kwargs):
     redirect(r"^about/leadership/boards-of-directors/$", "/about/leadership/#boards"),
     # Issue 13672
     redirect(r"/VendorDPA/?$", "https://assets.mozilla.net/pdf/VendorDPA.pdf", re_flags="i"),
+    # Issue 13754
+    redirect(r"rise-25/?$", "https://rise25.mozilla.org/"),
+    redirect(r"rise25/?$", "https://rise25.mozilla.org/"),
```
Comment (pmac) [inline bedrock/mozorg/redirects.py:574]: NIT: You can do this with one rule:

```python
redirect(r"rise-?25/?$", "https://rise25.mozilla.org/"),
```

**Review by pmac — APPROVED** [review]
Thanks! 👻

File: bedrock/mozorg/redirects.py (hunk 1 above)
Comment (nathan-barrett) [reply bedrock/mozorg/redirects.py:574]: nice, will add that and merge.

File: bedrock/mozorg/redirects.py [hunk 2]
```diff
@@ -569,4 +569,6 @@ def decider(request, **kwargs):
     redirect(r"^about/leadership/boards-of-directors/$", "/about/leadership/#boards"),
     # Issue 13672
     redirect(r"/VendorDPA/?$", "https://assets.mozilla.net/pdf/VendorDPA.pdf", re_flags="i"),
+    # Issue 13754
+    redirect(r"rise-?25/?$", "https://rise25.mozilla.org/"),
```
Comment (pmac) [inline bedrock/mozorg/redirects.py:573]: this should technically start with a `^` to ensure it doesn't match any other use of `rise25` in a URL. 
```suggestion
    redirect(r"^rise-?25/?$", "https://rise25.mozilla.org/"),
```

**Pull request merged**